Cubical/Algebra/Group/Free.agda

```diff
@@ -0,0 +1,487 @@
+{-# OPTIONS --safe #-}
```

Comment:
I think here or further down a comment explaining what will happen in the file would be great, i.e. something like "this file contains a construction of the free group using reduced words" or something like that.

Reply:
ok, wokring on that

Reply:
added brief summary